src/Annotator.js

```diff
@@ -149,6 +150,7 @@ class Annotator extends EventEmitter {
             .then(() => {
```

Comment:
Separate any actual changes to the lib as a separate commit

Reply:
Moved to this PR https://github.com/box/box-annotations/pull/167